spec/requests/admin/memberships_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Admin::MembershipsController, type: [:request, :admin] do
+  let(:organization) { create(:organization) }
+  let(:user) { create(:user) }
+
+  describe 'POST /admin/memberships' do
+    let(:create_params) do
+      {
+        user_id: user.id,
+        organization_id: organization.id,
+      }
+    end
+
+    it 'creates a membership' do
+      admin_post(
+        '/admin/memberships',
+        create_params,
+      )
+
+      aggregate_failures do
+        expect(response).to have_http_status(:success)
+        expect(json[:membership][:lago_user_id]).to eq(user.id)
```

Comment:
Can we also check that a membership has been created?
I mean, not only checking the json response